modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,155 @@
+** Removal of cloud event `PipelineResources` 
+* If the pipelines metrics feature does not work after a cluster upgrade, run the following command as a workaround:
++
+[source,terminal]
+----
+$ oc get tektoninstallersets.operator.tekton.dev | awk '/pipeline-main-static/ {print $1}' | xargs oc delete tektoninstallersets
+----
+* With this update, usage of external databases, such as the Crunchy PostgreSQL is not supported on {ibmpowerProductName}, {ibmzProductName}, and {linuxoneProductName}. Instead, use the default {tekton-hub} database. 
+
+[id="fixed-issues-1-10_{context}"]
+== Fixed issues
+
+* Before this update, the `opc pac` command generated a runtime error instead of showing any help. This update fixes the `opc pac` command to show the help message.
+* Before this update, running the `tkn pac create repo` command needed the webhook details for creating a repository. With this update, the `tkn-pac create repo` command does not configure a webhook when your GitHub application is installed.
+* Before this update, Pipelines as Code would not report a pipeline run creation error when Tekton Pipelines had issues creating the `PipelineRun` resource. For example, a non-existing task in a pipeline run would show no status. With this update, PAC shows the proper error message coming from `tekton/pipeline` along with the task that is missing.
+* This update fixes UI page redirection after a successful authentication. Now, you are redirected to the same page where you had attempted to log in to Tekton Hub.
+* This update fixes the `list` command with these flags, `--all-namespaces` and `--output=yaml`, for a cluster task, an individual task, and a pipeline.
+* This update removes the forward slash in the end of the `repo.spec.url` URL so that it matches with the URL coming from GitHub.
+* Before this update, the `marshalJSON` function would not marshal a list of objects. With this update, the `marshalJSON` function marshals the list of objects.
+* With this update, PAC lets you bypass GitHub enterprise questions in the CLI bootstrap GitHub application.
+* This update fixes the GitHub collaborator check when your repository has more than 100 users.
+* With this update, the `sign` and `verify` commands for a task or pipeline work fine without the kubernetes configuration file.
+* With this update, Tekton Operator cleans leftover pruner cron jobs if pruner has been skipped on a namespace.
+* Before this update, the API `ConfigMap` would not be updated with user configured value for catalog refresh interval. This update fixes the `CATALOG_REFRESH_INTERVAL` API in the Tekon Hub CR.
+* This update fixes reconciling of `PipelineRunStatus` when changing the `EmbeddedStatus` feature flag. This update resets the following parameters: 
+** Reset the `status.runs` and `status.taskruns` parameters to `nil` with `minimal EmbeddedStatus` 
+** Reset the `status.childReferences` parameter to `nil` with `full EmbeddedStatus`
```

Comment:
```suggestion
** The `status.childReferences` parameter to `nil` with `full EmbeddedStatus`
```
For both of these items, the lead-in sentence to the sub-list already says that these things are reset, so imo no need to repeat `reset` in the sub-list